### PR TITLE
[LIVE-11438][LIVE-11670] Remove feature flags "supportDeviceStax" and "staxWelcomeScreen"

### PR DESCRIPTION
**Pull request by ofreyssinet-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Support of Stax is _always_ enabled in LLM and LLD.
  - The video on the welcome screen of LLM is _always_ the Stax video.
  - The feature flag group "stax" does not exist anymore.

### 📝 Description

Cleanup of feature flags that have been activated in production.

### ❓ Context

- **JIRA or GitHub link**: [LIVE-11438] [LIVE-11670]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-11438]: https://ledgerhq.atlassian.net/browse/LIVE-11438?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-11670]: https://ledgerhq.atlassian.net/browse/LIVE-11670

### CODE REVIEW

**Review by mcayuelas-ledger — APPROVED** [review]
LGTM for Hub

File: apps/ledger-live-desktop/tests/userdata/speculos.json [hunk 1]
```diff
@@ -97,10 +97,6 @@
           "enabled": true,
           "overridesRemote": true
         },
-        "staxWelcomeScreen": {
-          "enabled": true,
-          "overridesRemote": true
-        },
```
Comment (valber-ledger) [inline apps/ledger-live-desktop/tests/userdata/speculos.json:103]: This file is not anymore on the repo, the new file is `speculos-tests-app.json` 
A rebase should resolve this issue 😃

**Review by valber-ledger — CHANGES_REQUESTED** [review]

File: apps/ledger-live-desktop/tests/userdata/speculos.json (hunk 1 above)
Comment (ofreyssinet-ledger) [reply apps/ledger-live-desktop/tests/userdata/speculos.json:103]: alright thanks 👍

Comment (ofreyssinet-ledger) [reply apps/ledger-live-desktop/tests/userdata/speculos.json]: @valber-ledger can you check again please?

**Review by valber-ledger — APPROVED** [review]
LGTM - just checked QAA files

**Review by valpinkman — APPROVED** [review]

**Pull request merged**